### DISCUSSION TITLE
支持Vue3吗？

### DISCUSSION THREAD

**Discussion by carveybunt** [antvis/Graphin#475] [Ideas]
[Graphin](https://github.com/antvis/Graphin) 很 `nice` ！但是我是用的是 Vue3 ，`Graphin` 能直接引入使用吗？

**Comment by pomelo-nwu** [comment]
暂时不支持，不过你可以试试 G6VP：https://github.com/antvis/G6VP
整个站点应用都通过一份sdk配置渲染生成，集成到vue中只需要用上微前端前端方案即可
